tests/unit/api/test_feature_list.py

```diff
@@ -1361,31 +1361,58 @@ def test_create_new_version__no_new_version_created(saved_feature_list):
     assert feature_list.id == saved_feature_list.id
 
 
+def create_feature_with_multiple_entity_relationships(event_view, feature_name):
+    """Create a feature with multiple entity relationships"""
+    feat_components = []
+    for group_by_key in ["col_int", "cust_id"]:
+        feat_name = f"sum_1d_{group_by_key}"
+        feat_component = event_view.groupby(group_by_key).aggregate_over(
+            value_column="col_float",
+            method="sum",
+            windows=["1d"],
+            feature_names=[feat_name],
+            feature_job_setting=FeatureJobSetting(
+                blind_spot="75m", frequency="30m", time_modulo_frequency="15m"
+            ),
+        )[feat_name]
+        feat_components.append(feat_component)
+
+    feat = feat_components[0] + feat_components[1]
+    feat.name = feature_name
+    return feat
```

Comment:
Revise this PR so that both `col_int` & `cust_id` entity are required in the feature.